pages/home.jsx

```diff
@@ -66,7 +67,7 @@ var Home = React.createClass({
         </div>
         <div className="white-background">
           <div className="centered content wide">
-          <h1>Join us next year!</h1>
+          <h1>Join Us at MozFest</h1>
```

Comment:
Changes in this page fixes some of what's in https://github.com/mozilla/mozillafestival.org/issues/816